src/patternfly/components/OptionsMenu/options-menu.scss

```diff
@@ -161,6 +162,19 @@
       --pf-c-options-menu__toggle--m-plain--Color: var(--pf-c-options-menu__toggle--m-plain--hover--Color);
     }
   }
+
+  &.pf-m-disabled,
+  &:disabled {
+    pointer-events: none;
+
+    &:not(.pf-m-plain) {
```

Comment:
Is there a conscious decision by interaction/design not to change the appearance of a plain options menu?

Reply:
@mceledonia @kybaker any thoughts on this? we keep dropdown / options menu / context selector the same visually for the most part. Are you guys ok with having the disabled styling the same?

Reply:
Yes I would think so! I definitely see dropdown and select as the same thing, to the user a select is just a multiselect dropdown. Context selector feels basically like a dropdown to me too but I'll defer to @kybaker there!

Reply:
@christiemolloy Agree with Michael here I to see them as the same thing.